github/github.go

```diff
@@ -478,12 +478,7 @@ func (c *Client) Do(ctx context.Context, req *http.Request, v interface{}) (*Res
 		return nil, err
 	}
 
-	defer func() {
-		// Drain up to 512 bytes and close the body to let the Transport reuse the connection
-		io.CopyN(ioutil.Discard, resp.Body, 512)
-		resp.Body.Close()
-	}()
-
+	defer resp.Body.Close()
```

Comment:
nit: I would like to see a blank line between the `defer` and the `newResponse`... since they aren't closely related.

...and while you're add it, I would like to tightly bind the defer to the `c.client.Do` above, so if you would like to get rid of the blank line above (line 480), that would be fantastic, IMHO.

Thanks, @diorahman!

Reply:
Ah OK. Thanks @gmlewis for the review. I updated it on https://github.com/google/go-github/pull/847/commits/d285239f10debdeece9710b1b15449b03bf9e41c.